setup.py

```diff
@@ -1,127 +1,170 @@
-log_files = [os.path.join('logs', j)
-             for j in os.listdir('logs')]
-exec(open('lib/jnpr/jsnapy/version.py').read())
-
-setup(name="jsnapy",
-      version=__version__,
-      description="Python version of Junos Snapshot Administrator",
-      author="Priyal Jain",
-      author_email="jnpr-community-netdev@juniper.net",
-      license="Apache 2.0",
-      keywords="Junos snapshot automation",
-      url="http://www.github.com/Juniper/jsnapy",
-      package_dir={'': 'lib'},
-      packages=find_packages('lib'),
-      package_data={
-           'jnpr.jsnapy': ['jsnapy.cfg', 'logging.yml', 'content.html'],
-      },
-      entry_points={
-          'console_scripts': [
-              'jsnapy=jnpr.jsnapy.jsnapy:main',
-          ],
-      },
-      scripts=['tools/jsnap2py'],
-      zip_safe=False,
-      install_requires=install_reqs,
-      data_files=[('/etc/jsnapy', ['lib/jnpr/jsnapy/logging.yml']),
-                  ('samples', example_files),
-                  ('/etc/jsnapy', ['lib/jnpr/jsnapy/jsnapy.cfg']),
-                  ('testfiles', ['testfiles/README']),
-                  ('snapshots', ['snapshots/README']),
-                  ('/var/log/jsnapy', log_files)
-                 ],
-      cmdclass={'install': OverrideInstall},
-      classifiers=[
-          'Environment :: Console',
-          'Intended Audience :: Developers',
-          'Intended Audience :: Information Technology',
-          'Intended Audience :: System Administrators',
-          'Intended Audience :: Telecommunications Industry',
-          'License :: OSI Approved :: Apache Software License',
-          'Operating System :: OS Independent',
-          'Programming Language :: Python',
-          'Programming Language :: Python :: 2.6',
-          'Programming Language :: Python :: 2.7',
-          'Topic :: Software Development :: Libraries',
-          'Topic :: Software Development :: Libraries :: Application Frameworks',
-          'Topic :: Software Development :: Libraries :: Python Modules',
-          'Topic :: System :: Networking',
-          'Topic :: Text Processing :: Markup :: XML'
-      ],
-      )
+#!/usr/bin/python
+
+# Copyright (c) 1999-2016, Juniper Networks Inc.
+#
+# All rights reserved.
+#
+
+import os, sys
+from os.path import expanduser
+from setuptools import setup, find_packages
+from setuptools.command.install import install
+import ConfigParser
+
+
+class OverrideInstall(install):
+    def run(self):
+
+        for arg in sys.argv:
+            if '--install-data' in arg:
+                break
+        else:
+            if hasattr(sys, 'real_prefix'):
+                self.install_data = os.path.join(os.environ.get('VIRTUAL_ENV'),'jsnapy')
+            elif 'win' in sys.platform:
+                self.install_data = os.path.join(os.path.expanduser('~'),'jsnapy')
+            else:
+                self.install_data = '/etc/jsnapy'
+
+        dir_path = self.install_data
+        mode = 0o777

```

Comment:
This was already done in the original setup file. Let me know if sure to make this change.